src/Services/NodePastDealsService.ts

```diff
@@ -2,6 +2,11 @@ import {NodePastDeal} from "../Models/NodePastDeal";
 
 export class NodePastDealsService {
 
+    public async replacePastDealsForNode(pastDeals: NodePastDeal[], nodeId: number) {
+        await NodePastDeal.destroy({where: {nodeId}});
+        return await NodePastDeal.bulkCreate(pastDeals);
+    }
+
     public async updateOrCreatePastDeal(
```

Comment:
This method is now never used.

Reply:
True but lets leave it in case we need it later (it's possible we change how deals are updated)